src/rust/catnap/win/transport.rs

```diff
@@ -15,9 +15,13 @@ use std::{
     net::SocketAddr,
     pin::Pin,
     time::Duration,
+    net::SocketAddrV4,
 };
 
-use libc::ENOTSUP;
+use libc::{
+    ENOTSUP,
+    EINVAL
+};
```

Comment:
Please, remove these imports and use `libc::ENOTSUP` and `libc::EINVAL` directly.